extensions/guacamole-auth-ldap/src/main/java/org/apache/guacamole/auth/ldap/user/AuthenticatedUser.java

```diff
@@ -53,6 +61,72 @@ public void init(Credentials credentials) {
         setIdentifier(credentials.getUsername());
     }
 
+    /**
+     * Get a map of attributes associated with this AuthenticatedUser.
+     *
+     * @return
+     *     The Map of arbitrary attributes associated with this
+     *     AuthenticatedUser object.
+     */
+    @Override
+    public Map<String, String> getAttributes() {
+        return attributes;
+    }
+
+    /**
+     * Sets a map of attributes associated with this AuthenticatedUser.
+     *
+     * @param attributes
+     *      A map of attribute key/value pairs to add to this AuthenticatedUser.
+     */
```

Comment:
Same thing as above - since we're overriding another already-documented method, here, you should be able to leave out the Javadoc comments.